app/scripts/migrations/134.test.ts

```diff
@@ -0,0 +1,56 @@
+import { migrate, version } from './134';
+
+const oldVersion = 133;
+
+describe(`migration #${version}`, () => {
+  it('updates the version metadata', async () => {
+    const oldStorage = {
+      meta: { version: oldVersion },
+      data: {},
+    };
+
+    const newStorage = await migrate(oldStorage);
+
+    expect(newStorage.meta).toStrictEqual({ version });
+  });
+
+  it('Does nothing if `usedNetworks` is not in the AppStateController state', async () => {
+    const oldState = {
+      AppStateController: {
+        timeoutMinutes: 0,
+      },
+    };
+
+    const transformedState = await migrate({
+      meta: { version: oldVersion },
+      data: { ...oldState },
+    });
+
+    expect(transformedState.data).toStrictEqual(oldState);
+  });
+
+  it('Removes `usedNetworks` from the AppStateController state', async () => {
+    const oldState: {
+      AppStateController: {
+        timeoutMinutes: number;
+        usedNetworks?: Record<string, boolean>;
+      };
+    } = {
+      AppStateController: {
+        timeoutMinutes: 0,
+        usedNetworks: {
+          '0x1': true,
+          '0x5': true,
+          '0x539': true,
+        },
+      },
+    };
+    const transformedState = await migrate({
+      meta: { version: oldVersion },
+      data: { ...oldState },
+    });
+
+    delete oldState.AppStateController.usedNetworks;
```

Comment:
This is a bit confusing because now the `oldState` variable no longer represents the "old state". Instead perhaps we can inline our expected state shape, or create a new variable called `expectedState` or something like that

Reply:
Indeed make sense